src/containers/TasksByDayPage/components/TabContainer/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+import Typography from '@material-ui/core/Typography'
+import styled from 'styled-components'
+
+export default function TabContainer(props) {
+  return (<StyledTypography component="div">{props.children}</StyledTypography>);
+}
+
+const StyledTypography = styled(Typography)`
+  padding: 0 24px 24px 24px;
```

Comment:
```suggestion
  padding: 0 24px 24px;
```
shorthand: top - right & left - bottom